### PR TITLE
Event schema & API

### PR DESCRIPTION
**Pull request by adamklingbaum**
- Adds schema for events and guests (**please review** the fields, types, validation, and required/not required in detail)
- Adds API documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,317 @@
...
+    "johnson family": [
+      {
+        "guestId": 12,
+        "firstName": "John",
+        "lastName": "Doe",
+        "email": "local@domain.com",
+        "rsvpStatus": "attending",
+        "rvspLastUpdated": "2021-05-23T20:17:46.384Z",
+        "rsvpNote": "I look forward to attending!",
+        "group": "joel family"
```
Comment (kyle-sablan) [inline README.md]: Might be a typo, but would a guest with group "joel family" be able to be listed under a different group, i.e. "johnson family"

File: README.md [hunk 2]
```diff
@@ -1 +1,317 @@
...
+        "firstName": "Jane",
+        "lastName": "Doe",
+        "email": "local@domain.com",
+        "rsvpStatus": "not attending",
+        "rvspLastUpdated": "2021-05-13T20:17:46.384Z",
+        "rsvpNote": "I will not be attending your wedding!",
+        "group": "johnson family"
+      }
+    ],
+    "family friends": [
```
Comment (kyle-sablan) [inline README.md]: Would the guests be split up into respective groups and guest objects wouldn't have a group property? Either works, but I noticed an inconsistency among guests in "johnson family" and those in "family friends"

**Review by kyle-sablan — CHANGES_REQUESTED** [review]
Everything looks good! Just added a couple some comments on the documentation for the API.

File: README.md (hunk 1 above)
Comment (adamklingbaum) [reply README.md]: Thanks for pointing this out. It was meant to be consistent but I missed some changes. All guests within a group would have that group's name as their `name` property. I will push up a revised version

File: README.md (hunk 2 above)
Comment (adamklingbaum) [reply README.md]: See above

**Review by hithisiskhai — APPROVED** [review]
Good to go.

**Review by hithisiskhai — APPROVED** [review]

**Review by kyle-sablan — APPROVED** [review]

**Pull request merged**